src/qibo/backends/clifford.py

```diff
@@ -540,19 +37,38 @@ def __init__(self, engine=None):
 
         from qibo.backends import _check_backend
 
-        engine = _check_backend(engine)
+        if engine is None:
+            engine = _check_backend(engine)
+            engine = engine.name if engine.platform is None else engine.platform
```

Comment:
Perfectly fine, but you could define directly in this module the `get_engine` function, and use that.

(you could even just call `engine()`, or if it sounds too ambiguous to you, then `resolve_engine()` - `get_` adds very little, but I might be too keen on this)